### ISSUE TITLE
About overview page

### ISSUE THREAD

**Issue by bruce1977** [leozdgao/em-doccenter#2]
Suggestion for overview page.
Add some **high priority** or the latest modified articles with short comments in overview page

**Comment by leozdgao** [comment]
@bruce1977 I haven't good idea currenly, let me consider about it.

**Comment by Rabbit0306** [comment]
Good idea! I think we can make the new knowledge articles and the most click articles in some block.
